### PR TITLE
Create ebates integration documentation based on embedded.

### PR DESCRIPTION
**Pull request by alexzhaobin**

### CODE REVIEW

File: docs/sdk/browser/android/ebates.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+The SDK acts as the middleman between the browser web view and the users Fillr profile which is managed via the embedded app.
+
+![Fillr integration components](https://docs.google.com/drawings/d/1YQfdm4CLklfGQHtGF_f3matpE1KA7tLw6hBOaFIlVJ0/pub?w=480&h=360)
+
+### Getting Started
+
+1. Add the following dependencies to the build.gradle file located in the application's directory (i.e. not the root build.gradle). Our repo is located on jcenter.
+
+   ```
+    compile(group: 'com.fillr', name: 'fillrcore', version: '3.1.5', ext: 'aar', classifier: 'fillrEmbeddedRelease') {
```
Comment (3000) [inline docs/sdk/browser/android/ebates.md]: This isnt up to date with the latest in prod? We are up to 4.3.8 i think...

File: docs/sdk/browser/android/ebates.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+       <fields>;
+   }
+   #bugsnag (if not)
+   -dontwarn com.bugsnag.**
+   -keep class com.bugsnag.** { *; }
+   -keepattributes SourceFile, LineNumberTable
+   ```
+
+
+### Importing user data from your app to Fillr
```
Comment (3000) [inline docs/sdk/browser/android/ebates.md]: Maybe we can just have this section as an extra to the original embedded doc?

I.e. just reference the embedded instructions from this ebates page like we reference the browser sdk from the embedded doc.

That way we wont have to duplicate a bunch of text and keep it in sync perhaps given there arent/shouldnt actually be that many differences between ebates and embedded?

File: docs/sdk/browser/android/ebates.md (hunk 1 above)
Comment (alexzhaobin) [inline docs/sdk/browser/android/ebates.md]: Will update

File: docs/sdk/browser/android/ebates.md (hunk 2 above)
Comment (alexzhaobin) [inline docs/sdk/browser/android/ebates.md]: @stuballs, updated, please review.

**Pull request merged**